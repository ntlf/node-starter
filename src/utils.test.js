import { add } from './utils';

it('should add 1 + 1', () => {
  const result = add(1, 1);

  expect(result).toMatchSnapshot();
});
