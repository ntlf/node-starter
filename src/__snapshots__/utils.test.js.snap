// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should add 1 + 1 1`] = `2`;
